app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
You can simplify this statement to:
```suggestion
        if not vstr.get("vaccine"):
```
It will check if the key exists in dict (otherwise `.get()` will return `None` by default), then if the values is set to `None` or either `False`. If it is, exception is raised